addOns/exim/src/main/java/org/zaproxy/addon/exim/automation/EximJob.java

```diff
@@ -0,0 +1,225 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.exim.automation;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.parosproxy.paros.CommandLine;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.control.Control;
+import org.zaproxy.addon.automation.AutomationData;
+import org.zaproxy.addon.automation.AutomationEnvironment;
+import org.zaproxy.addon.automation.AutomationJob;
+import org.zaproxy.addon.automation.AutomationProgress;
+import org.zaproxy.addon.automation.jobs.JobData;
+import org.zaproxy.addon.automation.jobs.JobUtils;
+import org.zaproxy.addon.exim.ExtensionExim;
+import org.zaproxy.addon.exim.log.LogsImporter;
+
+public class EximJob extends AutomationJob {
+
+    private static final String JOB_NAME = "import";
+    private static final String RESOURCES_DIR = "/org/zaproxy/addon/exim/resources/";
+
+    private static final String PARAM_TYPE = "type";
+    private static final String PARAM_FILE_NAME = "fileName";
+
+    private ExtensionExim extExim;
+
+    private Parameters parameters = new Parameters();
+    private Data data;
+
+    public EximJob() {
+        this.data = new Data(this, parameters);
+    }
+
+    private ExtensionExim getExtExim() {
+        if (extExim == null) {
+            extExim = Control.getSingleton().getExtensionLoader().getExtension(ExtensionExim.class);
+        }
+        return extExim;
+    }
+
+    @Override
+    public void verifyParameters(AutomationProgress progress) {
+        Map<?, ?> jobData = this.getJobData();
+        if (jobData == null) {
+            return;
+        }
+        JobUtils.applyParamsToObject(
+                (LinkedHashMap<?, ?>) jobData.get("parameters"),
+                this.parameters,
+                this.getName(),
+                null,
+                progress);
+    }
+
+    @Override
+    public void applyParameters(AutomationProgress progress) {
+        // Nothing to do
+    }
+
+    @Override
+    public Map<String, String> getCustomConfigParameters() {
+        Map<String, String> map = super.getCustomConfigParameters();
+        map.put(PARAM_TYPE, "");
+        map.put(PARAM_FILE_NAME, "");
+        return map;
+    }
+
+    @Override
+    public void runJob(AutomationEnvironment env, AutomationProgress progress) {
+
+        String type = this.getParameters().getType();
+        String fileName = this.getParameters().getFileName();
+
+        if (!StringUtils.isEmpty(fileName)) {
+            File file = new File(fileName);
+            if (file.exists() && file.canRead()) {
+                if (type.equalsIgnoreCase(EximJobDialog.TypeOption.HAR.getName())) {
```

Comment:
`name()`, to not use the localised names (which would not match once translated).